doc/concepts/decor.rst

```diff
@@ -287,6 +287,27 @@ See :ref:`modelgapjunctions`.
 4. Stimuli
 ~~~~~~~~~~
 
+A current stimulus is described by a *frequency* in Hertz and an amplitude *envelope*.
```

Comment:
I think it is unclear what the frequency actually _does_ here. Could you explain?
Maybe even drawing the resulting waveforms.

Reply:
I'm not going to draw a waveform :) But I can use terms like AC and DC to make this clearer.